src/main/kotlin/solve/menubar/view/MenuBarView.kt

```diff
@@ -1,19 +1,34 @@
 package solve.menubar.view
 
+import io.github.palexdev.materialfx.dialogs.MFXGenericDialog
+import io.github.palexdev.materialfx.dialogs.MFXStageDialog
+import javafx.geometry.Insets
 import solve.importer.controller.ImporterController
 import solve.importer.view.ImporterView
-import tornadofx.*
+import solve.main.MainView
+import solve.utils.MaterialFXDialog
+import tornadofx.View
+import tornadofx.action
+import tornadofx.button
+import tornadofx.hbox
 
 class MenuBarView : View() {
     val importer = find<ImporterView>()
+
+    var content = MFXGenericDialog()
+    var dialog = MFXStageDialog()
```

Comment:
Why not just create it once with correct parameters?

Reply:
you mean don't create separate content and dialog?